recipes/notion-calendar/webview-unsafe.js

```diff
@@ -0,0 +1,10 @@
+// Workaround for Notion's login window with Google OAuth
+if (window.location.href === 'https://calendar.notion.so/login') {
+  window.open = function (url) {
+    const newUrl = url.replace('popup=true', 'popup=false');
+    window.location.href = newUrl;
+  };
+  console.log(
+    'FERDIUM: Overriding window.open for Google OAuth login window in Notion.',
+  );
```

Comment:
We probably shouldnt merge it with `console.log()`?

Reply:
Yeah, we can take it out... It was just for debugging purposes (if there is an issue in the future regarding this).

But let's take it out 😁 thanks!